docs/content/latest/quick-start/install/macos.md

```diff
@@ -76,10 +76,15 @@ showAsideToc: true
 1. Each tablet maps to its own file, so if you experiment with a few hundred tables and a few tablets per table, you can soon end up creating a large number of files in the current shell. Make sure that this command shows a big enough value.
 
     ```sh
-    $ launchctl limit maxfiles
+    $ launchctl limit
     ```
 
-    We recommend setting the soft and hard limits to 1048576.
+    We recommend the following soft and hard limits.
+
+    ```output
+    maxproc     2500        2500
```

Comment:
My machine has a maxproc value higher than this (2666). Are you sure you want me to lower it?

Reply:
Changed the check to have at least minimum values